UnityProject/Assets/Scripts/PlayGroups/PlayerScript.cs

```diff
@@ -247,13 +251,13 @@ public ChatChannel GetAvailableChannels(bool transmitOnly = true)
 
         public ChatModifier GetCurrentChatModifiers()
         {
-			if (playerMove.isGhost)
-			{
-				return ChatModifier.None;
-			}
+            if (playerMove.isGhost)
+            {
+                return ChatModifier.None;
+            }
 
-			//TODO add missing modifiers
-			ChatModifier modifiers = ChatModifier.Drunk;
+            //TODO add missing modifiers
+            ChatModifier modifiers = ChatModifier.Drunk;

```

Comment:
The load of auto-corrected formatting, makes it harder to review your code...
Which is sad...